test/kubemark/start-kubemark.sh

```diff
@@ -49,17 +67,26 @@ gcloud compute instances create ${MASTER_NAME} \
 
 MASTER_IP=`gcloud compute instances describe hollow-cluster-master --zone=${ZONE} | grep networkIP | cut -f2 -d":" | sed "s/ //g"`
 
-until gcloud compute ssh --zone=${ZONE} hollow-cluster-master --command="ls" &> /dev/null; do
+until gcloud compute ssh --zone="${ZONE}" hollow-cluster-master --command="ls" &> /dev/null; do
   sleep 1
 done
 
-gcloud compute copy-files --zone=${ZONE} \
-  ${KUBE_ROOT}/_output/release-tars/kubernetes-server-linux-amd64.tar.gz \
-  ${KUBE_ROOT}/test/kubemark/start-kubemark-master.sh \
-  ${KUBE_ROOT}/test/kubemark/configure-kubectl.sh \
-  hollow-cluster-master:~
+if [ "${RUN_FROM_DISTRO}" == "false" ]; then
+  gcloud compute copy-files --zone="${ZONE}" \
+    "${KUBE_ROOT}/_output/release-tars/kubernetes-server-linux-amd64.tar.gz" \
+    "${KUBE_ROOT}/test/kubemark/start-kubemark-master.sh" \
+    "${KUBE_ROOT}/test/kubemark/configure-kubectl.sh" \
+    "hollow-cluster-master":~
+else
+  gcloud compute copy-files --zone="${ZONE}" \
+    "${KUBE_ROOT}/server//kubernetes-server-linux-amd64.tar.gz" \
+    "${KUBE_ROOT}/test/kubemark/start-kubemark-master.sh" \
+    "${KUBE_ROOT}/test/kubemark/configure-kubectl.sh" \
+    "hollow-cluster-master":~
```

Comment:
I can, but actually establishing connection is longer than copying small files when the connection is already established. It's faster this way, even if it's uglier.